universal-gateway/overview.mdx

```diff
@@ -1,58 +1,40 @@
 ---
 title: Universal Gateway Overview
 sidebarTitle: Overview
+description: Learn about ngrok's suite of common tools for building API and device gateways, identity-aware proxies, and site-to-site connectivity.
 ---
 
 ngrok's Universal Gateway is a suite of common tools for building API and device gateways, identity-aware proxies, and site-to-site connectivity.
 It secures, accelerates, and protects your applications.
-We call it a Universal Gateway because it supports [TCP](/universal-gateway/tcp/) and [TLS](/universal-gateway/tls/) applications as well as [HTTP/S](/universal-gateway/http/).
 
-## Concepts
+**What makes it "universal"?**
 
-Dive into the different parts of the Universal Gateway like Endpoints, Bindings, Pools, and Domains to understand how they work.
+The Universal Gateway can be used with software running locally or in the cloud, and with devices running on-premises or distributed in the field.
```

Comment:
This paragraph feels like it could benefit from being turned into a list, maybe with check icons ([mintlify-supported ones](https://lucide.dev/icons/?search=check))? The nature of the information is that it's a list of reasons why it's "universal", so the sentence structure kinda feels like a bulleted list that got turned into a paragraph imo